woocommerce/woocommerce
# In-app marketplace redesign phase 1

### All Submissions:


### Changes proposed in this Pull Request:

The Extend team wants to refresh the design of the in-app marketplace and better align it with the marketplace on WooCommerce.com. (Figma: seb2o1DExPxoh5d53ftJif-fi-812%3A1.)

This PR bundles the changes from several branches for phase 1 of the in-app marketplace rebuild. We're rendering the current "Browse Extensions" and "WooCommerce.com Subscriptions" tabs of the "Extensions" page as two separate pages, "Marketplace" and "My Subscriptions". "My Subscriptons" is pretty much unchanged, apart from that. "Marketplace" has a new header and restyled product cards.

We're hoping to build on these changes in phase 2 of the project, in which we'll rebuild the "Featured" section, add more data to the product cards, add a WooCommerce.com Helper connection element and other improvements.

#### PRs in this branch:

* [Split the Extensions page in WP Admin and add respective menu items](https://github.com/woocommerce/woocommerce/pull/30380)
* [New marketplace addons header](https://github.com/woocommerce/woocommerce/pull/30389)
* [Addressing PHPCS issues in Helper view file](https://github.com/woocommerce/woocommerce/pull/30405)
* [Add product card styling](https://github.com/woocommerce/woocommerce/pull/30410)
* [Layout tweaks](https://github.com/woocommerce/woocommerce/pull/30478)
* [Restyle Marketplace tab bar](https://github.com/woocommerce/woocommerce/pull/30464)
* [Update My Subscriptions page title](https://github.com/woocommerce/woocommerce/pull/30445)
* [In-app marketplace style tweaks](https://github.com/woocommerce/woocommerce/pull/30497)
* [Add Tracks event for new My Subscriptions page](https://github.com/woocommerce/woocommerce/pull/30488)
* [Display notices properly in Marketplace page](https://github.com/woocommerce/woocommerce/pull/30478)
* [In-app marketplace style tweaks](https://github.com/woocommerce/woocommerce/pull/30497)
* [Marketplace mobile layout fixes](https://github.com/woocommerce/woocommerce/pull/30498)
* [Add mobile section menu to marketplace](https://github.com/woocommerce/woocommerce/pull/30499)

Related WC Admin PR: [Split the Extensions page into Marketplace and My Subscriptions](https://github.com/woocommerce/woocommerce-admin/pull/7471)

### How to test the changes in this Pull Request:

These are general steps for the overall change – please see the constituent PRs if you want to make more detailed checks.

1. Check out this branch.
2. Notice the WooCommerce > Extensions menu item has been replaced by WooCommerce > Marketplace and WooCommerce > My Subscriptions.
3. Please disregard the "Extensions" title in the breadcrumb on these two pages – we're addressing that in a WooCommerce Admin [PR](https://github.com/woocommerce/woocommerce-admin/pull/7471).
4. View the My Subscriptions page. You should see your store's connection to WooCommerce.com as usual, or connect it.
5. View the Marketplace page. It should appear like the designs on all viewports. You should be able to browse the different sections of the marketplace and make searches as normal. Clicking on a product's "View details" button should take you to the product page. Promotions should appear as normal in the Featured section and (depending on your locale and plugin setup) in the Shipping and Payments sections.

### Other information:


### Changelog entry

Enhancement - Split the "Extensions" page into two pages, "Marketplace" and "My Subscriptions". Apply new design to Marketplace.

### Screenshots

<img width="500" src="https://user-images.githubusercontent.com/1647564/129351097-5f61c74d-171a-4361-a0a8-869ae0cb8396.png">

<img width="250" src="https://user-images.githubusercontent.com/1647564/129363255-f94f18e8-9385-4d0e-9e80-30bab63b4be2.png">
